finders/gps.py

```diff
@@ -1,20 +1,32 @@
+"""
+Module for GPS data
+"""
 from dataclasses import dataclass
-from math import sin, cos, acos, pi, radians
+from math import sin, cos, acos, radians
+
 
 @dataclass
 class GPSPosition:
     latitude: float
     longitude: float
 
-def gps_distance(Start : GPSPosition,End : GPSPosition):
+
+def gps_distance(start: GPSPosition, end: GPSPosition) -> float:
```

Comment:
@freddyfok The return value would be a float not an int 
```suggestion
def gps_distance(start: GPSPosition, end: GPSPosition) -> float:
```